requirements.txt

```diff
@@ -1,5 +1,4 @@
 koji
-rpm-py-installer
```

Comment:
1. This change is not related to the PR.
2. I believe `rpm-py-installer` is actually needed for the project currently.
3. CI would likely need a workaround like this: https://github.com/release-engineering/ubi-manifest/commit/dd877cabd7e0ac14c54bd79302eaf733aea1fd8a but please submit that in different PR.

Reply:
ok, i will submit another patch to fix the CI